sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1242,6 +1242,13 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val REMOVE_REDUNDANT_SORTS_ENABLED = buildConf("spark.sql.execution.removeRedundantSorts")
```

Comment:
We need this config? cc: @cloud-fan

Reply:
If we want to backport this, it's safer to have a config for the new rule, in case it has bugs.